bingo/postgres/src/pg_common/bingo_pg_common.h

```diff
@@ -37,155 +39,170 @@ class BingoPgCommon {
-//   static void* getIndexData(BingoPgBuffer& pg_buffer, int& data_len);
+    //   static void formIndexTuple(BingoPgBuffer& buffer, void* map_data, int size);
+    //   static void* getIndexData(BingoPgBuffer& pg_buffer, int& data_len);
 
     static void printBitset(const char* name, BingoPgExternalBitset& bitset);
     static void printFPBitset(const char* name, unsigned char* bitset, int size);
 
-
-//   static char* getTextData(PG_OBJECT text_datum, int& size);
+    //   static char* getTextData(PG_OBJECT text_datum, int& size);
 
     static void setDefaultOptions();
-//   static dword getFunctionOid(const char* name, indigo::Array<dword>& types);
-//   static dword getFunctionOid1(const char* name, dword type1);
-
-//   static dword callFunction(dword oid, indigo::Array<dword>& args);
-//   static dword callFunction1(dword oid, dword arg1);
-
+    //   static dword getFunctionOid(const char* name, indigo::Array<dword>& types);
+    //   static dword getFunctionOid1(const char* name, dword type1);
 
+    //   static dword callFunction(dword oid, indigo::Array<dword>& args);
+    //   static dword callFunction1(dword oid, dword arg1);
 
-//   static void executeQuery(const char* query_str);
+    //   static void executeQuery(const char* query_str);
     static int executeQuery(indigo::Array<char>& query_str);
-    static int executeQuery(const char *format, ...);
-    static bool tableExists(const char* schema_name,const char* table_name);
+    static int executeQuery(const char* format, ...);
+    static bool tableExists(const char* schema_name, const char* table_name);
 
-    static void createDependency(const char* schema_name,const char* index_schema, const char* child_table, const char* parent_table);
+    static void createDependency(const char* schema_name, const char* index_schema, const char* child_table, const char* parent_table);
     static void dropDependency(const char* schema_name, const char* index_schema, const char* table_name);
     static void appendPath(const char* schema_name);
 
     static char* releaseString(const char* str);
 
-
-
     /*
      * This class allows to avoid save-load sequence errors
      * You can save or load structures by calling one method
      * Numbers, arrays, double arrays etc are can be handled by this helper
      */
-    class DataProcessing {
+    class DataProcessing
+    {
     public:
-
-        template <typename T>
-        static void handleNumber(T& number, indigo::Scanner* scanner, indigo::Output* output) {
+        template <typename T> static void handleNumber(T& number, indigo::Scanner* scanner, indigo::Output* output)
+        {
             _handleNumber(number, scanner, output);
         }
 
-        template <typename T>
-        static void handleArray(indigo::Array<T>& data, indigo::Scanner* scanner, indigo::Output* output) {
+        template <typename T> static void handleArray(indigo::Array<T>& data, indigo::Scanner* scanner, indigo::Output* output)
+        {
```

Comment:
IMHO, this is wrong formatting change. 
`template<>` should be allowed to take own line since the other part is long.

Reply:
fixed